src/databricks/labs/ucx/assessment/queries/00_3_count_external_locations.sql

```diff
@@ -0,0 +1,4 @@
+-- viz type=counter, name=Storage Locations, counter_label=Storage Locations, value_column=count_total
```

Comment:
the idea there that it's not a real external location, but can change that, of course

Reply:
This widget should have some sort of description then, or we should include the dashboard sketch in the README to avoid confusion with External Location and Storage Credentials

Reply:
Strong +1 on explaining the dashboards somewhere! Could be in a separate PR though (and I guess once we stabilize the dashboard changes soon)?